code/modules/mob/logout.dm

```diff
@@ -7,14 +7,18 @@
 	create_log(MISC_LOG, "Logged out")
 	// `holder` is nil'd out by now, so we check the `admin_datums` array directly
 	//Only report this stuff if we are currently playing.
-	if(GLOB.admin_datums[ckey] && SSticker && SSticker.current_state == GAME_STATE_PLAYING)
+	if(GLOB.admin_datums[ckey] && SSticker?.current_state == GAME_STATE_PLAYING)
```

Comment:
Unneeded. SSticker can never be null. Remove the check entirely.

Reply:
What about if someone logs out before SSticker has loaded? Right as the game's first starting.

Reply:
Subsystems aren't bought into existence on Initialize(). 

They exist as soon as you press go on dreamdaemon.

This check only exists because of the old legacy code that didn't generate the ticker until you pressed start ingame, which is about 10 years old.

Reply:
Ah okay, that makes sense. 👍